e3sm_to_cmip/__main__.py

```diff
@@ -149,9 +157,7 @@ def __init__(self, args: Optional[List[str]] = None):
         logger.info(f"    * precheck_path='{self.precheck_path}'")
         logger.info(f"    * freq='{self.freq}'")
         logger.info(f"    * realm='{self.realm}'")
-        logger.info(f"    * Writing log output file to: {log_filename}")
-
-        self.handlers = self._get_handlers()
+        # logger.info(f"    * Writing log output file to: {log_filename}")
```

Comment:
That was my question:  You would need to have found the logfile, in order to read the comment that the log output was being written to the file you are reading.  Does that make sense?  Is the idea that the logfile may later be moved, and you want a record of where it was when it was being written (full-path)?  Otherwise, it seems oddly self-referential.

Reply:
Thanks for explaining. That makes sense to me.